### DISCUSSION TITLE
April 2024 Meeting

### DISCUSSION THREAD

**Discussion by azukov** [PyORBIT-Collaboration/PyORBIT3#9] [General]
To be held in the end of April, 2024.

**Comment by Eothred** [comment]
Agenda includes

- Status of meson build (Sasha, all)
- Minerva use case (Lennert and/or Jarno)
- Open issues and merge requests
- AOB

Meeting scheduled Thursday 25th at 3pm CET.

**Reply by Eothred** [reply]
Here's what I noted down during the meeting, please feel free to add anything I missed or noted incorrectly.

- Lennert presented Minerva/Myrrha introduction
	- Myrrha is a Belgian ADS project
	- Minerva is the first stage which will consist of one normal-conducting front-end and SCL to bring beam energy to 140 MeV. Around 100m total length.
	- Started on a Myrrha Linac Factory that reads straight from TraceWin files, inspired by SNS Linac Factory
	- Unsure which RF gap model to use
	- Unsure how to deal with TTF for their translation from TraceWin
	- Brandon commented that  they use BaseRF gap mostly
- Isabella observes some SC differences at 5GeV between PyORBIT and TraceWin (transfer line studies)
- @Eothred  commented that there is a recently opened #12 pull request.
	- Been tested to work on one flavour of Linux, and OSX. 
	- Others are encouraged to test it and report back if it does not work.
- Unknown if there was progress on #8 since last meeting.
- #4 is put on hold, as it is probably irrelevant once meson build is working.

Next meeting tentatively scheduled for Thursday May 30th, at 9am EDT/3pm CET.
- Aim to merge meson build proposal
